jdxsmith/rancid-tomatillos
# Clean up files to be readable, and remove trailing coma for genre list.

#### What’s this PR do?
* Cleans up code for better developer empathy
* Removes trailing coma for the genre list

#### Where should the reviewer start?
* `src/App.js` line 3
* `src/App.test.js` line 9
*  `src/Movie-form.js` line 25
* `src/Movie-form.test.js` line 12
* `src/Movies.js` line 2
* `src/Poster.css` line 39
* `src/Poster.js` line 6

#### How should this be manually tested?
* Pull down and open in your text editor, and run `npm start`
  * Note that when you click on a movie, the trailing comma is now gone

#### Any background context you want to provide?
* Refer to Turing styles guide

#### What are the relevant tickets?
* Closes #28 
* Closes #32
